src/Microsoft.AspNetCore.SignalR.Core/IHubClients`T.cs

```diff
@@ -5,24 +5,73 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A clients caller abstraction for a hub.
+    /// </summary>
+    /// <typeparam name="T">The client caller type.</typeparam>
     public interface IHubClients<T>
     {
+        /// <summary>
+        /// Gets a caller to all hub clients.
+        /// </summary>
+        /// <returns>A client caller.</returns>
         T All { get; }
 
+        /// <summary>
+        /// Gets a caller to all hub clients excluding the specified client connections.
```

Comment:
`all hub clients...` -> `all clients connected to the hub, excluding the specified connections`